Shamaz/ViewController.swift

```diff
@@ -10,24 +10,68 @@ import UIKit
 
 class ViewController: UIViewController {
 
-    let test1 = "hello"
+    @IBOutlet weak var futureButton: UIButton!
+    @IBOutlet weak var pastButton: UIButton!
+    @IBOutlet weak var storytellingTask: UILabel!
+    
+    let appTextColor = UIColor(white: 1.00, alpha: 1.00)
+    let appLabelFont = UIFont(name: "Chalkduster", size: 30.0)
+    let appButtonFont = UIFont(name: "Chalkduster", size: 20.0)
+    let appButtonRadius:CGFloat = 5.00
+    let appLightGreenColor = CGColor(srgbRed: 0.25, green: 0.80, blue: 0.60, alpha: 1.00)
+    let appLightBlueColor = CGColor(srgbRed: 0.13, green: 0.77, blue: 0.80, alpha: 1.00)
     
-    let test2 = "world"
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        futureButton.layer.cornerRadius = appButtonRadius
+        futureButton.layer.backgroundColor = appLightGreenColor
```

Comment:
I don't know why, but we usually set the background on the view itself instead of its layer:
`futureButton.backgroundColor = .green`